tests/apps/core/test_management_commands_create_demo_site.py

```diff
@@ -173,6 +191,7 @@ def test_command_create_demo_site(
 
         # Check that the calls to create organizations and persons were triggered as
         # expected
+        self.assertEqual(mock_course.call_count, NB_COURSES)
         self.assertEqual(mock_organization.call_count, NB_ORGANIZATIONS)
         self.assertEqual(mock_subject.call_count, NB_SUBJECTS)
```

Comment:
If you want to make it more robust to catch yesterday's bug we can test here the args with which the mock was called.

Reply:
It should be subject of another PR since it seems a little bit complicated, also i saw that test on `mock_page.call_args_list` at line 199 does not work, in fact the loop does not iter on anything because of slice `[root_pages_length:]` that returns an empty list. Removing the slice also fail because `expected_call` is not correct.

Reply:
You're right it is not testing anything since we moved the page creation to the factories for page extension!

I'm going to prepare a fix commit.